js/functions.js

```diff
@@ -39,6 +39,29 @@ var appLoaded = false;
 var changes = false;
 var noCrowdingGeneralText = "";
 var noCrowdingLocationText = "";
+var appStage;
```

Comment:
set default value to appStage in case that there is no such file env.json

Reply:
done.